src/test/resources/features/99d_employment_qa_smoke.feature

```diff
@@ -19,8 +19,8 @@ Feature: Smoke tests for end of year Employment Journeys to DES/IF in QA - Incom
     When the user clicks the continue button
     Then the user should be redirected to the "Your Income Tax Return" page
     When the user clicks on the employment link
-    Then the user should be redirected to the "Employment" page
-    When the user clicks on the change link
+    Then the user should be redirected to the "PAYE employment" page
+    When the user clicks on the view employment details link
```

Comment:
I think this would be the change employer selector instead

Reply:
i shall double check

Reply:
good spot, updated.